package.json

```diff
@@ -15,10 +15,8 @@
   },
   "homepage": "https://github.com/formidablelabs/karma-benchmarkjs-reporter",
   "scripts": {
-    "start": "watch 'npm run build' src/",
-    "prepublish": "npm test && npm run build",
-    "test": "eslint src/",
-    "build": "babel src -d build"
+    "prepublish": "npm test",
```

Comment:
Can we maybe switch this to `preversion` and use an `npm version`-based workflow?

There's a serious bug with `npm prepublish` that npm `wontfix`-ed: https://github.com/npm/npm/issues/3059